js/views/modals/purchase/Purchase.js

```diff
@@ -452,6 +452,7 @@ export default class extends BaseModal {
         singleSelect: true,
         selectFirst: true,
         radioStyle: true,
+        useCache: false,
```

Comment:
@cpacia is the one that suggested this PR, and using the non-cached mods. 

If we don't want to use them, then we just close this PR, and everything is good.